service/src/test/java/bio/terra/cbas/dependencies/sam/TestSamService.java

```diff
@@ -64,15 +72,24 @@ void init() throws ApiException {
         .thenAnswer(
             (Answer<UserStatusInfo>)
                 invocation -> {
-                  if (bearerToken != null
-                      && bearerToken.getToken().equals(tokenWithCorrectAccess)) {
-                    return mockUser;
-                  } else if (bearerToken != null
-                      && bearerToken.getToken().equals(tokenCausingInterrupt)) {
-                    throw new InterruptedException();
-                  } else {
-                    // expired or no token
-                    throw new ApiException(401, "Unauthorized :(");
+                  if (bearerToken == null) {
```

Comment:
Cleaned this up to include the remaining tokens and be more explicit about our test cases